velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -280,4 +280,43 @@ struct CombinationsFunction {
   }
 };
 
+template <typename T>
+struct udf_array_sum {
+  VELOX_DEFINE_FUNCTION_TYPES(T)
+  template <typename OT, typename IT>
+  FOLLY_ALWAYS_INLINE bool call(OT& out, const IT& array) {
```

Comment:
use 'void' return type since this function never produces nulls

Reply:
Done.